nuxt.config.ts

```diff
@@ -18,8 +17,13 @@ export default defineNuxtConfig({
   ],
   nitro: {
     prerender: {
-      crawlLinks: false
-    }
+      crawlLinks: true,
+      routes: ['/en/', '/nl/'],
+    },
+  },
+  routeRules: {
+    '/**': { static: true },
+    '/*/team/**': { prerender: false },
```

Comment:
`prerender` is only documented at the Nuxt docs not at Nitro, not sure why.

Reply:
Can you add that as a comment so other developers will understand this as well after this PR is merged?

Reply:
We could link to the Nuxt docs but the types should explain it too.